src/components/map-controls.js

```diff
@@ -282,8 +279,8 @@ export default class MapControls extends PureComponent {
     if (yDelta > 0) {
       // Dragging downwards, gradually decrease pitch
       if (Math.abs(this.props.height - startPos[1]) > PITCH_MOUSE_THRESHOLD) {
-        const scale = yDelta / (this.props.height - startPos[1]);
-        pitch = (1 - scale) * PITCH_ACCEL * startPitch;
+        const scale = yDelta / (this.props.height - startPos[1]) * PITCH_ACCEL;
```

Comment:
While I don't doubt this behaves better, this still looks weird to me, is `scale` still the right name after this change (if it ever was) or is there something more intuitive.